### PR TITLE
Allow optional disabling develop packages

### PR DESCRIPTION
**Pull request by tiberiuichim**
You can set a "develop": false value to any package, to keep a package in mrs-developer.json, but disable its processing. This makes it work more like the "sources" of zc.buildout/mrs.developer.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+module.exports = {
+    'env': {
+        'browser': true,
```
Comment (pnicolli) [inline .eslintrc.js:3]: Is ``browser`` needed? This should be a node-only package.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# 1.6.0 (unreleased)
+
+- Allow setting a package as non-develop by setting `"develop": false` in
+  mrs-developer.json, for any package configuration.  @tiberiuichim
```
Comment (pnicolli) [inline CHANGELOG.md:4]: Just a typo, filename should be ``mrs.developer.json``

File: README.md [hunk 3]
```diff
@@ -57,7 +57,18 @@ By using the `local` property, we can declare a path that will be added in `tsco
 
 By running the `missdev` command, those repositories will be checked out in the `./src/develop` folder and they will be added into the `tsconfig.json` file in the `paths` property, so the compiler will use them instead of the `node_modules` ones.
 
-Existing `paths` entries will be preserved if they do not parget a folder located in `src/develop`.
+Existing `paths` entries will be preserved if they do not target a folder located in `src/develop`.
+
+It is possible to keep a package in mrs-developer.json, but don't process it, by setting `develop: false`. This allows an easier development workflow of those packages, as they can be easier toggled between dev and released modes.
```
Comment (pnicolli) [inline README.md:62]: Same as above, filename should be ``mrs.developer.json``

**Review by pnicolli — COMMENTED** [review]
+1 for the proposed feature.
I'm not a super fan of having both new features and internal changes like the addition of eslint or dependency upgrades in the same PR, as it confuses the PR diff a little. I would suggest separating those PRs, unless the upgrades are really needed for the feature.

File: .eslintrc.js (hunk 1 above)
Comment (tiberiuichim) [reply .eslintrc.js:3]: Yeah, ah... missed that. I though I've passed "node" as preset to eslint --init, but then it put browser from inspect of existing code. Blah

**Comment by tiberiuichim** [comment]
> I'm not a super fan of having both new features and internal changes like the addition of eslint or dependency upgrades in the same PR, as it confuses the PR diff a little. I would suggest separating those PRs, unless the upgrades are really needed for the feature.

The PR is not big and the package doesn't see a lot of activity. Actually I also need to add a test, I didn't manage to add one. But if I remove the eslint thing, I'd have to edit the code without proper js integration in my editor. That's not a place I want to go :-)

**Review by sneridagh — APPROVED** [review]
LGTM

**Review by ebrehault — APPROVED** [review]

**Comment by ebrehault** [comment]
Thanks!
1.6.0 is released

**Pull request merged**